services/scientist/maana-physical-quantity.md

```diff
@@ -564,14 +847,55 @@ schema { query: Query }
 | Example | Multiplication and division |
 | :--- | :--- |
 | Info | When you pass arguments that have different units or physical dimensions, computer algebra on units of measure and physical dimensions automatically constructs the correct unit of measure for the returned physical quantity |
-| Query | { product\(factors: \[{parseFrom: "12 inch"}, {parseFrom: "1 Hz"}, {parseFrom: "1/3 acre"}\], precision: 2\) {  name,  unitOfMeasure {  dimension {  name}}}} |
-| Response | { "data": { "product": { "name": "4.0 Hz·acre·in", "unitOfMeasure": { "dimension": { "name": "VOLUMETRIC\_FLOW"}}}}} |
+| Query | { product\(factors: \[{parseFrom: "12 inch"}, {parseFrom: "1 Hz"}, {parseFrom: "1/3 acre"}\], precision: 2\) {  name,  unitOfMeasure {  dimension {  formula}}}} |
+| Response | { "data": { "product": { "name": "4.0 Hz·acre·in", "unitOfMeasure": { "dimension": { "formula": "L³T⁻¹"}}}}} |
 
 ### Size Comparisons
 
 | Example | Comparing size without converting units |
 | :--- | :--- |
 | Info | The physical quantity service will perform the necessary unit conversions for you, as long as the units are dimensionally consistent |
-| Query | { extract\(sources: \["1 inch","1 foot","1 yard"\]\) {  entity{  isGT\(rhs:{parseFrom:"2 1/2 feet"}\), isGTE\(rhs:{parseFrom:"2.54 centimeters"}\), isEqual\(rhs:{parseFrom:"3 feet"}\), isLT\(rhs:{parseFrom:"0.5 meters"}\), isLTE\(rhs:{parseFrom:"1/5280 mile"}\)}}} |
-| Response | { "data": { "extract": \[ { "entity": { "isGT": false, "isGTE": true, "isEqual": false, "isLT": true, "isLTE": true}}, { "entity": { "isGT": false, "isGTE": true, "isEqual": false, "isLT": true, "isLTE": true}}, { "entity": { "isGT": true, "isGTE": true, "isEqual": true, "isLT": false, "isLTE": false}}\]}} |
+| Query | query REFERENCE{ parse(source:"35 cm") { ... PhysicalQuantityFragment }}<br>query EXAMPLE($REF:PhysicalQuantityInput!){extractBatch(sources: ["1 inch","1 foot","1 yard"]) { entity{  isGT(rhs:$REFERENCE)}}} |
```

Comment:
Used REF in variable def, then REFERENCE in query

Reply:
Good get.   Fixed it.